wallet/src/lib.did

```diff
@@ -136,7 +167,7 @@ service : {
   // Cycle Management
   wallet_balance: () -> (record { amount: nat64 }) query;
   wallet_send: (record { canister: principal; amount: nat64 }) -> (WalletResult);
-  wallet_receive: () -> ();  // Endpoint for receiving cycles.
+  wallet_receive: (opt ReceiveOptions) -> ();  // Endpoint for receiving cycles.
```

Comment:
The call works from existing callers. The `dfx canister call` behavior you wrote up before only happens because DFX is inspecting the DID file; Candid has no problem with a function accepting additional optional parameters, so if your tool isn't checking candid files at runtime (like the CDK) or has an outdated candid file, it will succeed.

Reply:
Is there a plan to remove `wallet_receive`? It is not strictly needed.